src/textual/_spatial_map.py

```diff
@@ -24,17 +33,17 @@ def __init__(self, grid_width: int = 100, grid_height: int = 20) -> None:
         """
         self._grid_size = (grid_width, grid_height)
         self.total_region = Region()
-        self._map: defaultdict[tuple[int, int], list[ValueType]] = defaultdict(list)
+        self._map: defaultdict[GridCoordinate, list[ValueType]] = defaultdict(list)
         self._fixed: list[ValueType] = []
 
-    def _region_to_grid(self, region: Region) -> Iterable[tuple[int, int]]:
+    def _region_to_grid_coordinate(self, region: Region) -> Iterable[GridCoordinate]:
```

Comment:
Plural? As I understand it, multiple coordinates can be returned, right?

```suggestion
    def _region_to_grid_coordinates(self, region: Region) -> Iterable[GridCoordinate]:
```